examples/direct_fidelity_estimation.py

```diff
@@ -207,13 +208,14 @@ def direct_fidelity_estimation(circuit: cirq.Circuit, qubits: List[cirq.Qid],
     Args:
         circuit: The circuit to run the simulation on.
         qubits: The list of qubits.
-        noise: The noise model when doing a simulation.
+        sampler: Either a noisy simulator or an engine.
         n_trial: The total number of Pauli measurements.
         n_clifford_trials: In case the circuit is Clifford, we specify the
             number of trials to estimate the noise-free pauli traces.
-        samples_per_term: is set to 0, we use the 'noise' parameter above and
+        samples_per_term: is set to 0, we use the 'sampler' parameter above as
+            a noise (must be of type cirq.DensityMatrixSimulator) and
             simulate noise in the circuit. If greater than 0, we ignore the
-            'noise' parameter above and instead run an estimation of the
+            'sampler' parameter above and instead run an estimation of the
```

Comment:
If `samples_per_term` is greater than zero, we don't actually ignore `sampler` - we use it in the course of estimation of the characteristic function (to obtain the values of the Pauli observables involved).

Reply:
Done.